### DISCUSSION TITLE
Estructura del pseudocodigo en el parcial

### DISCUSSION THREAD

**Discussion by francorojo** [dds-jv/foro#172] [Consultas generales] [answered]
Buenas, tenia una duda, para la realizacion del pseudocodigo, prefieren que sea directo en el word y mostrar el avance o se puede hacer en el ide (sin necesidad que compile ni testearlo) y luego ir pasando de a partes con los colores del formato para que quede mas legible/lindo visualmente?
Graciaas!

**Comment by gastonprieto** [comment, marked as answer]
Buenas @francorojo !

Mañana vamos a pasar al grupo un lineamiento sobre el formato de entrega del parcial y las cosas a definir.

Gaston

**Comment by francorojo** [comment]
Graciaaas!
